### PR TITLE
0x06a - Hardware Security & Secure Boot - minor linguistic issues

### PR DESCRIPTION
**Pull request by AnnaSzk**
Corrected minor language issues and improved readability

### CODE REVIEW

File: Document/0x06a-Platform-Overview.md [hunk 1]
```diff
@@ -28,18 +28,22 @@ The [iOS security architecture](https://www.apple.com/business/docs/iOS_Security
...
+When an iOS device is powered on, it reads the initial instructions from the read-only Boot ROM, which bootstraps the system. This memory contains immutable code, together with Apple Root CA, which is etched in the silicon die during fabrication process, creating the root of trust. During the next step, the Boot ROM code checks if the signature of the iBoot bootloader is correct. Once the signature is validated, the iBoot checks the signature of the next boot stage, which is iOS kernel. If any of these steps fail, the boot process is immediately terminated and the device enters the recovery mode and displays "Connect to iTunes" screen. However, if the Boot ROM fails to load, the device enters a special low level recovery mode, which is called Device Firmware Upgrade (DFU). This is the last resort to recover the device to its original state. In this case the device will have no sign of activity, i.e. its screen will not display anything.
+
+This entire process is called "Secure Boot Chain". It aims at ensuring that the systems and its components are written and distributed by Apple. The Secure Boot chain consists of kernel, bootloader, kernel extension and baseband firmware.
```
Comment (sushi2k) [inline Document/0x06a-Platform-Overview.md]: systems => system

File: Document/0x06a-Platform-Overview.md [hunk 2]
```diff
@@ -28,18 +28,22 @@ The [iOS security architecture](https://www.apple.com/business/docs/iOS_Security
 
 #### Hardware Security
 
-The iOS security architecture makes heavy use of hardware-based security features that enhance overall performance and security. Each device comes with two built-in AES 256-bit keys, UID and GID, fused/compiled into the application processor and Secure Enclave during manufacturing. There is no way to directly read these keys through software or debugging interfaces such as JTAG. Encryption and decryption operations are performed by hardware AES crypto-engines with exclusive access to the keys.
+iOS security architecture makes heavy use of hardware-based security features that enhance overall performance and security. Each iOS device comes with two built-in AES 256-bit keys – GID and UID – fused and compiled into the application processor and Secure Enclave during manufacturing. There is no direct way to read these keys by a software or debugging interfaces such as JTAG. Encryption and decryption operations are performed by hardware AES crypto-engines having exclusive access to these keys.
```
Comment (Sjord) [inline Document/0x06a-Platform-Overview.md]: > *The* iOS security

I think leaving "The" here would be better.

> read these keys by *a* software

Software is an uncountable noun, so you can't have *a* software.

File: Document/0x06a-Platform-Overview.md [hunk 3]
```diff
@@ -28,18 +28,22 @@ The [iOS security architecture](https://www.apple.com/business/docs/iOS_Security
...
+When an iOS device is powered on, it reads the initial instructions from the read-only Boot ROM, which bootstraps the system. This memory contains immutable code, together with Apple Root CA, which is etched in the silicon die during fabrication process, creating the root of trust. During the next step, the Boot ROM code checks if the signature of the iBoot bootloader is correct. Once the signature is validated, the iBoot checks the signature of the next boot stage, which is iOS kernel. If any of these steps fail, the boot process is immediately terminated and the device enters the recovery mode and displays "Connect to iTunes" screen. However, if the Boot ROM fails to load, the device enters a special low level recovery mode, which is called Device Firmware Upgrade (DFU). This is the last resort to recover the device to its original state. In this case the device will have no sign of activity, i.e. its screen will not display anything.
```
Comment (Sjord) [inline Document/0x06a-Platform-Overview.md]: > displays "Connect to iTunes" screen

I think this still needs an article, i.e. "a" or "the".

File: Document/0x06a-Platform-Overview.md (hunk 1 above)
Comment (AnnaSzk) [inline Document/0x06a-Platform-Overview.md]: OK

File: Document/0x06a-Platform-Overview.md (hunk 2 above)
Comment (AnnaSzk) [inline Document/0x06a-Platform-Overview.md]: Corrected

File: Document/0x06a-Platform-Overview.md (hunk 3 above)
Comment (AnnaSzk) [inline Document/0x06a-Platform-Overview.md]: OK

**Comment by sushi2k** [comment]
@AnnaSzk Can you please commit your changes? Then we can merge the PR. Thanks

**Pull request closed without merging**